grouper/fe/template_util.py

```diff
@@ -4,104 +4,78 @@
 from dateutil.relativedelta import relativedelta
 from jinja2 import Environment, PackageLoader, select_autoescape
 from pytz import UTC
-from six import string_types
 
 from grouper.fe.settings import settings
 
 if TYPE_CHECKING:
-    from typing import Any, Callable, Dict, Optional, Union
+    from typing import Any, Callable, Dict, Optional
 
+# Components of a relativedelta, in order from longest interval to shortest.
+DELTA_COMPONENTS = ["year", "month", "day", "hour", "minute", "second"]
```

Comment:
Guessing we're not including microseconds for human readability? If so, a comment mentioning that would make the test case more obvious (see below)